### ISSUE TITLE
Case yjms80qh missing payout tx

### ISSUE THREAD

**Issue by stakoholic** [bisq-network/support#1446]
Description:

Missing final payout from above trade id. from 12 April , 23

 I wanted to withdraw funds from bisq and the tx is not showing up in the mempool for few days, so i made a resync and after the resync i recognized that the payout utxo from above trade, which i wanted to withdraw dont exist aynmore in my funds, and after a little research, it seems the payment from this trade yjms80qh-42eabc08-4318-4d2a-ad28-704579c6313b-199 was not payed out to me , but wallet show me correct funds so i don´t recognized it till i made the resync now

Bisq version:

actually 1.9.10

Device or machine:
linux manjaro

![bisq1](https://github.com/bisq-network/support/assets/42588964/f02f61a2-29d6-4a14-b824-8d67fe8d0087)

**Comment by ghost** [comment]
Yep, that has funds still locked in the multisig.  I've DM'd @pazza83 and @leo816.  Hopefully you can contact support on Matrix to get immediate assistance.

**Comment by pazza83** [comment]
Hi @stakoholic 

Please can you copy and paste the following:

- maker transaction ID
- taker transaction ID
- deposit transaction ID

Many thanks

**Comment by leo816** [comment]
please post that information and I will look into it

**Comment by stakoholic** [comment]
Hi @pazza83 @leo816 

yeah sure, here it is 

maker txid   : 611bece7db265f140e474829c6f4eb4b000ed399292a2429254648fe42684abe
taker txid     : 11adfaedd938db6d3b80a8fb9de40b7378e66751e3b0ff6df768a683d636d163
deposit txid : 3cae3df47bc03b1603d6d14c241777217537f34476dcf91c4974cfca4b4e02f2

thank you for looking at this case

**Comment by pazza83** [comment]
Hi @stakoholic 

Yes the funds are locked for this trade:

https://mempool.space/tx/3cae3df47bc03b1603d6d14c241777217537f34476dcf91c4974cfca4b4e02f2

Are you able to open mediation for this trade?

If not please click on 'Detail Data' in the button on your screen shot and let me know who is the assigned mediator for this trade. Just search for mediator and let me know if it is Leo or myself.

**Comment by stakoholic** [comment]
@pazza83 

no mediation is no longer possible, since the trade is finished since month and therefore in the history.

in the detail data, the selected mediator is: leo816

**Comment by leo816** [comment]
I have re-opened the ticket and waiting for a response from the seller to make a manual payout

**Comment by stakoholic** [comment]
@leo816 alrigth, should i see the ticket in the support?? because the trade is still in my history and what is when the seller will not respond??

**Comment by leo816** [comment]
@stakoholic I will need your delayedpayout tx either from your logs or from your closedtrades files, could you please send it to me on matrix? https://bisq.wiki/Dumping_delayed_payout_transactions

**Comment by darawhelan** [comment]
The funds for this trade have paid out to the burning men address.

This issue can be closed.
